src/main/java/com/google/codeu/data/Datastore.java

```diff
@@ -97,11 +97,8 @@ public void storeMessage(Message message) {
 
         Message message = new Message(id, user, text, timestamp);
         messages.add(message);
-      } catch (Exception e) {
-        System.err.println("Error reading message.");
-        System.err.println(entity.toString());
-        e.printStackTrace();
       }
+      finally {}
```

Comment:
Having a try block with no catch block and an empty finally does not do anything. If the code compiles, then in this case it looks like the code might not throw any exceptions, so you should remove the `try {` entirely.